include/ultra64/abi.h

```diff
@@ -294,13 +294,20 @@ typedef short ENVMIX_STATE[40];
         _a->words.w1 = (u32)(a2);                                    \
 }
 
-#define aClearBuffer(pkt, d, c)                                         \
-{                                                                       \
-        Acmd *_a = (Acmd *)pkt;                                         \
-                                                                        \
-        _a->words.w0 = _SHIFTL(A_CLEARBUFF, 24, 8) | _SHIFTL(d, 0, 24); \
-        _a->words.w1 = (u32)(c);                                        \
-}
+/*
+ * Clears DMEM data by writing zeros.
+ *
+ * @param pkt pointer to a Acmd command
+ * @param dmemAddr DMEM address to be cleared
+ * @param size number of bytes to clear (rounded up to 16 byte alignment)
```

Comment:
```suggestion
 * @param size number of bytes to clear (rounded up to next multiple of 16)
```
I prefer to say multiple when talking about sizes, and alignment when talking about addresses.
I'm not sure whether this should be qualified with "in microcode" or similar, since it may be confusing to anyone just reading this file as to where the rounding happens.

Reply:
fwiw reading this I would assume the number of bytes should be rounded up in the command, not that the microcode does it

Reply:
It could also be argued that it's more of a side-effect of the implementation than something programmed intentionally. The specific implementation is a loop that decrements by 16 bytes in each iteration, the loop will continue while the size is larger than 0. For a multiple of 16, this obviously will reach exactly 0. For a non-multiple of 16, it will end at some negative number and clear up to the next multiple.

Reply:
maybe `(the actual number of bytes cleared is rounded up to a multiple of 16)`, or make it a `@note` :thinking: ?